source/process/partner-programs.rst

```diff
@@ -15,19 +15,25 @@ The purpose of the Mattermost Authorized Reseller Program is to enable customers
 
 Companies who enter into a Mattermost Authorized Reseller agreement typically have existing relationships with customers and help them procure information technology products. 
 
-Purchasing as a Mattermost Authorized Reseller 
+Becoming a Mattermost Authorized Reseller 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- Complete a `contact form <https://about.mattermost.com/contact/>`_ with Mattermost requesting an authorized reseller discussion, including the name of the customer to which you would like to resell. 
-- You'll work with a partner manager over email, or possibly phone, to understand the details of your business and the resale transaction. 
+- Complete a `contact form <https://about.mattermost.com/contact/>`_ with Mattermost requesting an authorized reseller discussion and, if available, include the name of the customer to which you would like to resell. 
+- You'll be in contact with a partner manager over email, or possibly phone, to understand the details of your business and the resale transaction. 
 - Once the resale is approved internally, you will be provided a one-time click-sign authorized reseller agreement to complete, along with purchase order information for your customer. 
 
-   - Note: Mattermost does not accept customer purchase orders or custom terms. 
+Placing an Order as a Mattermost Authorized Reseller 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  When ready to place an order submit a PO to sales@mattermost.com along with the name, address and contact information of your customer.  
```

Comment:
Insert a comma after "order"

Should we use abbreviations (e.g. PO) in public facing documents?  Suggest replacing with "purchase order"